src/components/SelectCol.js

```diff
@@ -40,8 +40,8 @@ class SelectCol extends React.Component {
             borderRadius: "50%",
             border:"none",
             position:"fixed",
```

Comment:
Can you have a try on the position relative, so see when user scroll down, will the button follow?